libr/core/cmd_debug.c

```diff
@@ -250,7 +250,8 @@ static const char *help_msg_dm[] = {
 
 static const char *help_msg_dmi[] = {
 	"Usage: dmi", "", " # List/Load Symbols",
-	"dmi", "[libname] [symname]", "List symbols of target lib",
+	"dmi", "[j|q|*] [libname] [symname]", "List symbols of target lib",
+	"dmia", "[j|q|*] [libname] [symname]", "List all info of target lib",
```

Comment:
I tried the patch and it looks good, however I'm not sure why `dmia` should accept a `symname` as well. Isn't that just a duplicate of `dmi`? I would remove the `symname` argument from `dmia` and let it just print all info. If a user wants to reduce the output he can anyway use `~`.